cmp/relativetimestamp/RelativeTimestamp.js

```diff
@@ -83,166 +83,71 @@ class LocalModel {
     refreshDisplay() {
         this.display = getRelativeTimestamp(this.timestamp, this.options);
     }
-
 }
 
 
 /**
  * Returns a string describing the approximate amount of time between a given timestamp and the
  * present moment in a friendly, human readable format.
  *
- * @param {(Date|int)} timestamp - Date or milliseconds representing the starting / reference time.
+ * @param {(Date|int)} timestamp - Date or milliseconds representing the starting time / time to compare.
```

Comment:
I see that we had `int` before but please swap to `number` as I don't believe `int` is a valid jsdoc type (please correct me if I am wrong)